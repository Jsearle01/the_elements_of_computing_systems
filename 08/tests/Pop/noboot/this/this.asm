@THIS
D=M
@temp
M=D
@SP
M=M-1
A=M
D=M
@temp
A=M
M=D
@12345
@THIS
D=M+1
@temp
M=D
@SP
M=M-1
A=M
D=M
@temp
A=M
M=D
@12345
@THIS
D=M
@2
D=D+A
@temp
M=D
@SP
M=M-1
A=M
D=M
@temp
A=M
M=D
