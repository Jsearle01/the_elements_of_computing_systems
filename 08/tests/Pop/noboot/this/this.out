| RAM[0] | RAM[1] | RAM[2] | RAM[3] | RAM[4] | RAM[5] | RAM[6] | RAM[7] |RAM[102]|RAM[103]|RAM[104]|RAM[105]|RAM[106]|RAM[107]|
|    256 |    102 |    103 |    104 |    105 |      5 |      6 |      7 |    102 |    103 |     58 |     57 |     56 |    107 |
